tests/psp34/psp34.tests.ts

```diff
@@ -0,0 +1,174 @@
+import { expect, setupContract, fromSigner } from '../helpers'
+
+describe('MY_PSP34', () => {
+  async function setup() {
+    return setupContract('my_psp34', 'new')
+  }
+
+  async function setup_receiver() {
+    return setupContract('psp34_receiver', 'new')
+  }
+
+  // TODO: add test for collection_id
```

Comment:
You still can add the test to check that it fails right night

Reply:
makes sense
will add one and change the TODO comment as well